backend/apps/sandbox/views.py

```
@@ -582,3 +611,105 @@ def _normalize(text):
+    DialogueNodeSerializer,
+)
+
+
+class ModerationScenarioViewSet(viewsets.ReadOnlyModelViewSet):
+    """
+    Viewset for Moderation Scenarios (De-escalation Trainer).
+    """
+
+    queryset = ModerationScenario.objects.all().order_by("-created_at")
+    serializer_class = ModerationScenarioSerializer
+    permission_classes = [permissions.AllowAny]
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)